src/main/kotlin/app/revanced/patches/reddit/ad/banner/HideBannerPatch.kt

```diff
@@ -4,7 +4,13 @@ import app.revanced.patcher.data.ResourceContext
 import app.revanced.patcher.patch.ResourcePatch
 import app.revanced.patcher.patch.annotation.Patch
 
-@Patch(description = "Hides banner ads from comments on subreddits.")
+@Patch(
+    description = "Hides banner ads from comments on subreddits.",
+    // Constrained to last working version (2024.0.17) until https://github.com/iBotPeaches/Apktool/issues/3534
+    // is addressed which is causing crashes during patching.
+    // See https://github.com/ReVanced/revanced-patches/issues/3099.
+    compatiblePackages = [CompatiblePackage("com.reddit.frontpage", ["2024.0.17"])]
```

Comment:
```suggestion
    // Constrained to last working version (2024.17.0) until https://github.com/iBotPeaches/Apktool/issues/3534
    // is addressed which is causing crashes during patching.
    // See https://github.com/ReVanced/revanced-patches/issues/3099.
    compatiblePackages = [CompatiblePackage("com.reddit.frontpage", ["2024.17.0"])]
```
Wasn't the last working version `2024.17.0` and not `2024.0.17`?